libecs/include/pivot/ecs/Core/Event/manager.hxx

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "pivot/ecs/Core/Data/value.hxx"
+#include "pivot/ecs/Core/Event/description.hxx"
+#include "pivot/ecs/Core/Systems/manager.hxx"
+#include "pivot/ecs/Core/types.hxx"
+
+#include <any>
+#include <vector>
+
+namespace pivot::ecs::event
+{
+
+/** \brief Manages all the events in a Scene
+ *
+ * The Manager send event and execute systems listing this event
+ */
+class Manager
```

Comment:
Why making a class that doesn't server any purpose other than calling another class ? Can you move the `sendEvent` method into the systems::Manager ?

Reply:
This will be corrected in the refactor of the event manager. It should attribute indexes to events, and store the indexes of the systems listening to each event.